spyder/config/main.py

```diff
@@ -426,7 +427,7 @@
               'editor/run cell and advance': RUN_CELL_AND_ADVANCE_SHORTCUT,
               # -- In plugins/editor.py
               'editor/show/hide outline': "Ctrl+Alt+O",
-              'editor/show/hide project explorer': "Ctrl+Alt+P",

```

Comment:
I think this shortcut is no more needed because

1. If Spyder has no project active, the Project explorer is empty.
2. If the project is active, I think users would expect the Project explorer to be visible all the time.

@goanpeca, what do you think?